website/src/views/PageStateHandler.ts

```diff
@@ -209,9 +209,14 @@ export abstract class CompareVariantsStateHandler<ColumnData extends BaselineAnd
     }
 
     public addEmptyFilter(pageState: CompareVariantsData<ColumnData>): CovidCompareVariantsData {
-        const lastId = Math.max(...Array.from(pageState.filters.keys()));
+        const newId = () => {
+            if (pageState.filters.size === 0) {
+                return 0;
+            }
+            return Math.max(...Array.from(pageState.filters.keys())) + 1;
+        };
```

Comment:
```suggestion
        const newId = pageState.filters.size === 0 ? 0 : Math.max(...Array.from(pageState.filters.keys())) + 1;
```

Reply:
I had it like this and then moved to the function to make it clearer. But since you also prefer this, I changed it back. :)